src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs

```diff
@@ -304,6 +347,49 @@ public string CreateAnswer(Bundle bundle)
             return answer;
         }
 
+        /// <summary>
+        /// Creates SDP answer asynchronously with option to an offer received from a remote peer.
+        /// </summary>
+        /// <remarks>The WebRTC must be in the <see cref="WebRTCState.Negotiating"/></remarks>
+        /// <returns>The SDP answer.</returns>
+        /// <exception cref="InvalidOperationException">The WebRTC is not in the valid state.</exception>
+        /// <exception cref="ObjectDisposedException">The WebRTC has already been disposed.</exception>
+        /// <seealso cref="CreateAnswer(Bundle)"/>
+        /// <since_tizen> 9 </since_tizen>
+        public async Task<string> CreateAnswerAsync() => await CreateAnswerAsync(null);
+
+        /// <summary>
+        /// Creates SDP answer with option asynchronously with option to an offer received from a remote peer.
+        /// </summary>
+        /// <remarks>The WebRTC must be in the <see cref="WebRTCState.Negotiating"/></remarks>
+        /// <param name="bundle">Configuration options for the answer.</param>
+        /// <returns>The SDP answer.</returns>
+        /// <exception cref="InvalidOperationException">The WebRTC is not in the valid state.</exception>
+        /// <exception cref="ObjectDisposedException">The WebRTC has already been disposed.</exception>
+        /// <seealso cref="CreateAnswer()"/>
+        /// <since_tizen> 9 </since_tizen>
+        public async Task<string> CreateAnswerAsync(Bundle bundle)
+        {
+            ValidateWebRTCState(WebRTCState.Negotiating);
+
+            var bundle_ = bundle?.SafeBundleHandle ?? new SafeBundleHandle();
+
+            var tcsSdpCreated = new TaskCompletionSource<string>();
+
+            NativeWebRTC.SdpCreatedCallback cb = (handle, sdp, _) =>
+            {
+                tcsSdpCreated.TrySetResult(sdp);
+            };
+
+            NativeWebRTC.CreateSDPAnswerAsync(Handle, bundle_, cb, IntPtr.Zero).
+                    ThrowIfFailed("Failed to create answer asynchronously");
+
+            var answer = await tcsSdpCreated.Task.ConfigureAwait(false);
+            await Task.Yield();
+
+            return answer;
+        }
+
```

Comment:
Similar situation on the synchronous methods(CreateAnswer/Offer(bundle)),
Could it be removed in API 9? or at least, something has to be added in description.

Reply:
I'll remove the API with `Bundle`.